pipert/core/message_handlers.py

```diff
@@ -71,3 +131,9 @@ def connect(self):
 
     def close(self):
         self.conn.close()
+
+    def _add_offset_to_stream_id(self, stream_id, offset):
+        fixed_id = stream_id.split("-")
+        last_msg_id_to_read = (fixed_id[0] + "-" +
+                               str(int(fixed_id[1]) + offset))
```

Comment:
Maybe `'-'.join([fixed_id[0], str(int(fixed_id[1] + offset))])` will be nicer?

Reply:
i agree, i will change it.